base/linalg/bidiag.jl

```diff
@@ -154,10 +153,29 @@ for func in (:A_ldiv_B!, :Ac_ldiv_B!, :At_ldiv_B!)
         B
     end
 end
-for func in (:A_ldiv_Bt!, :Ac_ldiv_Bt!, :At_ldiv_Bt!)
+function A_ldiv_Bt!(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix)
+    mA,nA = size(A)
+    tmp = similar(B[:, 2])
+    m, n = size(B)
+    if mA != n || nA != m
+        throw(DimensionMismatch())
+    end
+    nm = n*m
+    for i = 1:size(B, 1)
+        copy!(tmp, 1, B, i:m:nm, n)
+        ($func)(A, tmp)
```

Comment:
I think it is tested on lines 49,50 of the `test/ilnalg/bidiag.jl` file? I separated `A \ B^T` from the others because the dimension boundary checking is different.